charts/gsp-cluster/charts/gsp-monitoring/dashboards/concourse-internal.json

```diff
@@ -0,0 +1,553 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "editable": true,
+  "gnetId": null,
+  "graphTooltip": 1,
+  "id": null,
```

Comment:
is it ok for this `id` to be `null`?

I can't remember if it's required, the other dashboards appear to have a number set.

Reply:
I... don't know, but I did manage to :green_apple:  this board manually earlier and it worked

Reply:
The [documentation doesn't say much useful](https://grafana.com/docs/grafana/latest/http_api/dashboard/#identifier-id-vs-unique-identifier-uid), and the example they give for create leaves it `null` ... worst case; we end up seeing some duplicated dashboards and will know what to fix, best case we discover we don't have to come up with random non-clashing id's in the future.

Let's just give it a go :+1: